cp2/ControlPanel/ControlPanel/models.py

```diff
@@ -44,12 +44,12 @@ def user_created_handler(sender, instance, *args, **kwargs):
         return
 
     demoinfo_editor, _ = api_post(
-        "/api/demoinfo/editor", "get", data={"email": new_editor.email}
+        f"/api/demoinfo/editor?email={new_editor.email}", "get"
     )
     new_editor_exists = demoinfo_editor.get("editor", None)
 
     demoinfo_editor, _ = api_post(
-        "/api/demoinfo/editor", "get", data={"email": old_editor.email}
```

Comment:
shouldn't you use `params={"email"..}` here, instead of constructing the query string yourself?

Reply:
Works both ways, since it is only one param in that call I put it in the URL

Reply:
the issue is escaping special characters and potential injections